README.md

```diff
@@ -4,7 +4,31 @@ Next Gen docos using VuePress.
 
 ## Local development
 
+### Prerequisites
+
+1. `NodeJs` installed (see [installation guide](https://nodejs.org/en/download//)).
+2. `Yarn` installed (see [installation guide](https://classic.yarnpkg.com/en/docs/install/)).
+3. For Windows development `Windows Subsystem for Linux` (it's neeeded because internally [sed](https://www.gnu.org/software/sed/manual/sed.html) is used in build scripts)\
+   - [WSL 2 installation guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+   - [Windows Terminal](https://www.microsoft.com/pl-pl/p/windows-terminal/9n0dx20hk701?rtc=1&activetab=pivot:overviewtab)
+
+### Running docs locally
+
 1. Clone the repo
 2. Run `yarn install`
 3. Run `yarn docs:dev`
 
+### Getting external samples
```

Comment:
I don't think we need this as we plan to change the workflow

Reply:
Removed in https://github.com/EventStore/documentation/pull/163/commits/b48cca4b907699b1cb5bb21347f39fd0ed4855b3.